### PR TITLE
os/filestore: include journal in get_devices() result

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
@hsiang41

File: src/os/filestore/FileJournal.cc [hunk 1]
```diff
@@ -2198,3 +2198,18 @@ void FileJournal::get_devices(set<string> *ls)
     get_dm_parents(dev_node, ls);
   }
 }
+
+void FileJournal::collect_metadata(map<string,string> *pm)
+{
+  BlkDev blkdev(fd);
```
Comment (hsiang41) [inline src/os/filestore/FileJournal.cc]: char partition_path[PATH_MAX];
  char dev_node[PATH_MAX];

File: src/os/filestore/FileJournal.cc [hunk 2]
```diff
@@ -2185,3 +2185,16 @@ off64_t FileJournal::get_journal_size_estimate()
   dout(20) << __func__ << " journal size=" << size << dendl;
   return size;
 }
+
+void FileJournal::get_devices(set<string> *ls)
+{
+  char dev_node[PATH_MAX];
+  BlkDev blkdev(fd);
+  if (int rc = blkdev.wholedisk(dev_node, PATH_MAX); rc) {
+    return rc;
```
Comment (hsiang41) [inline src/os/filestore/FileJournal.cc]: "return rc" not need.

**Comment by liewegas** [comment]
updated, i think it's all working now.  (sorry about that, i was working on my laptop and wasn't about to build test anything)

**Comment by hsiang41** [comment]
Great! Verification result is good.

**Pull request merged**